### PR TITLE
updates readme

### LINKED ISSUE #5430: update sponsors in readme

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #5430

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,18 +64,17 @@ Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com
...
-<a href="https://opencollective.com/openemr/sponsor/6/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/6/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/7/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/7/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/8/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/8/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/9/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/9/avatar.svg"></a>
+Support this project on OpenCollective. [[Become a sponsor](https://opencollective.com/openemr#sponsor)]
+
+<a href="https://opencollective.com/openemr#sponsors" target="_blank"><img src="https://opencollective.com/openemr/sponsors.svg?width=890"></a>
+
+<a href="https://github.com/sponsors/openemr" target="_blank">
+  <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-heart icon-sponsor mr-1 v-align-middle color-fg-sponsors anim-pulse-in">
```
Comment (bradymiller) [inline README.md:72]: is there supposed to be something here that shows up in the readme?

Comment (bradymiller) [inline README.md:72]: is there supposed to be something here that shows up in the readme?

Comment (bradymiller) [reply README.md:72]: sorry for the echo. github has been acting odd today

Comment (stephenwaite) [reply README.md:72]: it should show this little heart which shows up in my local
Screenshot from 2022-06-08 12-04-31

**Review by bradymiller — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stephenwaite) [reply README.md:72]: might not appear on github, thanks for noticing

Comment (stephenwaite) [reply README.md]: maybe these latest changes are too harsh and it was nice to see some avatars at the bottom but adding the yml for funding hopefully organizes it better

File: FUNDING.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+custom:             ["https://www.open-emr.org/donate/"]
+open_collective:    openemr
+#bitcoin: 3GCNLbiZmHP26fA77NkLgY4tKdEts3ADQg
+#ethereum: 0xcD7542b2DcF41072aC7783C7cbc31B0f1E257E80
```
[the remark this answers is not in the document]
Comment (stephenwaite) [reply FUNDING.yml:5]: hopefully this works

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

**Pull request merged**